.pre-commit-config.yaml

```diff
@@ -20,10 +20,15 @@ repos:
             CHANGELOG-dev.md
           )$
         always_run: false # required to respect exclude
-  - repo: https://github.com/psf/black
-    rev: 22.3.0
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 25.1.0
     hooks:
       - id: black
+        # It is recommended to specify the latest version of Python
+        # supported by your project here, or alternatively use
+        # pre-commit's default_language_version, see
+        # https://pre-commit.com/#top_level-default_language_version
+        language_version: python3.13
```

Comment:
This is all very well, but you'll need to adjust the workflow that runs this to adapt to it. Or not specify a language version here because the people who are recommending that are _not really supporting multiple language versions_. They might claim they are, but they aren't.